tests/common/util/TestObject.cpp

```diff
@@ -89,6 +89,8 @@ createLedgerHeader(std::string_view ledgerHash, ripple::LedgerIndex seq, std::op
     ledgerHeader.seq = seq;
 
     if (age) {
+        // Note: be cautios of using small age values as the underlying NetClock precision is seconds
```

Comment:
```suggestion
        // Note: be cautious of using small age values as the underlying NetClock precision is seconds
```

😆

Reply:
Just using a large value won't help - if you provide 10000 and 10001 as values, the comparison will still be flake.
The idea is to provide values which are not close to each other.

Reply:
Why precommit hook didn't catch it?

Reply:
https://github.com/crate-ci/typos?tab=readme-ov-file#why-was--not-corrected

> typos maintains a list of known typo corrections to keep the false positive count low so it can safely run unassisted.

> This is in contrast to most spell checking UIs people use where there is a known list of valid words

I think it might not be in the corrections list.

Reply:
I checked `typos --files` and this test file shows up (as expected).
And looked at the `words.csv` here: https://raw.githubusercontent.com/crate-ci/typos/refs/heads/master/crates/typos-dict/assets/words.csv
Haven't found `cautios` there.

Reply:
Nice one, thanks!